ironfish-rust/src/keys/public_address.rs

```diff
@@ -16,92 +16,76 @@ use std::{convert::TryInto, io};
 
 use super::{IncomingViewKey, SaplingKey};
 
-/// The address to which funds can be sent, stored as a diversifier and public
-/// transmission key. Combining a diversifier with an incoming_viewing_key allows
-/// the creation of multiple public addresses without revealing the viewing key.
-/// This allows the user to have multiple "accounts", or to even have different
-/// payment addresses per transaction.
+/// The generator for public address.
+pub const PUBLIC_KEY_GENERATOR: SubgroupPoint = SubgroupPoint::from_raw_unchecked(
```

Comment:
It's likely best not expose the generator as part of the public api

```suggestion
pub(crate) const PUBLIC_KEY_GENERATOR: SubgroupPoint = SubgroupPoint::from_raw_unchecked(
```

Reply:
Remove the generator point value here. Import from zkp package instead.